docs/content/server/new_blocks.md

```diff
@@ -109,64 +122,46 @@ To create a new block, follow these steps:
    __all__ = ["ai", "sample", "reddit", "text", "AVAILABLE_BLOCKS", "wikipedia", "discord", "get_wikipedia_summary"]

```

Comment:
This is no longer needed. the block will be auto-discovered once you add some classes in the `blocks` folder